inotify/src/lib.rs

```diff
@@ -33,6 +33,7 @@ use std::os::unix::ffi::OsStrExt;
 use std::os::unix::io::RawFd;
 use std::path::Path;
 use std::slice;
+use std::collections::HashMap;
```

Comment:
You should be able to remove this. Deriving traits shouldn't require any imports.

The compiler also [warns about this import being unused](https://travis-ci.org/hannobraun/inotify-rs/jobs/257566812#L260).